docs/docs/api/producer.md

```diff
@@ -48,44 +55,43 @@ Engine when they meet following 2 conditions:
      [Observe](/docs/api/observe) is the only way to add a trigger to a
      `producer`. Whenever an `Observe`d value is changed (e.g using
      [Update](/docs/api/update)), the `producer` is re-executed.
-     !!! Not necessarily, you can also create an empty producer that will be called when the Engine boots but it won't receive any argument.
-
-
-## Other ideas:
 
-Producers are managed entirely by the Engine which will instantiate, trigger and remove them.
+> Note: You can also create an empty producer that will be executed only once when the Engine boots up
```

Comment:
As mentioned above, it's also true that a producer can be void of `Observe`s but still have other `Get` or `Update` statements in the header.